### PR TITLE
Start the server awareness

### PR DESCRIPTION
**Pull request by brichet**
This PR start the server awareness periodic update.

It follows https://github.com/jupyter-server/pycrdt/pull/178.

### CODE REVIEW

**Comment by davidbrochart** [comment]
Let me know if you need help with this (I see tests are failing).

**Comment by brichet** [comment]
Yes, I took a look at the tests, it seems that `test_yroom_restart` never ends.

The tests pass if we remove the [error](https://github.com/jupyter-server/pycrdt/blob/5df30e00abefb411918667d9110765af2911ce7e/python/pycrdt/_awareness.py#L65-L66) if the awareness is already started. This probably means that the awareness is not properly stopped, but I don't know how to fix it.
And I'm not sure if we need to raise in this case.

File: pycrdt_websocket/yroom.py [hunk 1]
```diff
@@ -231,6 +231,7 @@ async def start(
                         self._task_group.start_soon(self._stopped.wait)
                         self._task_group.start_soon(self._watch_ready)
                         self._task_group.start_soon(self._broadcast_updates)
+                        self._task_group.start_soon(self.awareness.start)
```
Comment (davidbrochart) [inline pycrdt_websocket/yroom.py:234]: You need that above too, in the `if from_context_manager:` branch.

File: pycrdt_websocket/yroom.py [hunk 2]
```diff
@@ -240,6 +241,7 @@ async def stop(self) -> None:
         if self._task_group is None:
             raise RuntimeError("YRoom not running")
         self._stopped.set()
+        self.awareness.stop()
```
Comment (davidbrochart) [inline pycrdt_websocket/yroom.py:244]:
```suggestion
        await self.awareness.stop()
```

**Review by davidbrochart — CHANGES_REQUESTED** [review]

**Comment by brichet** [comment]
Thanks @davidbrochart

**Review by davidbrochart — APPROVED** [review]
Thanks @brichet.

**Pull request merged**